api/internal/impl/bootstrap/bootstrap.go

```diff
@@ -38,9 +37,15 @@ import (
 //   - keeperID: The unique identifier of the target Keeper
 //
 // Returns:
-//   - nil if successful
-//   - error if marshaling, validation, network request, or server-side
-//     processing fails
+//   - *sdkErrors.SDKError: nil on success, or one of the following errors:
+//   - ErrSPIFFENilX509Source: if source is nil
+//   - Errors from net.Post(): if the HTTP request fails (e.g., ErrAPINotFound,
+//     ErrAccessUnauthorized, ErrAPIBadRequest, ErrStateNotReady,
+//     ErrNetPeerConnection)
+//
+// Note: The function will fatally crash (via log.FatalErr) for unrecoverable
+// errors such as marshal failures (ErrDataMarshalFailure) or invalid
+// contribution length (ErrCryptoInvalidEncryptionKeyLength).
```

Comment:
This is for security. -- Bootstrap is a critical operation: Better to crash the container and restart than leaving the system in a potentially insecure/indeterminate state.